app/view_models/sign_up_completions_show.rb

```diff
@@ -0,0 +1,39 @@
+class SignUpCompletionsShow
+  def initialize(loa3_requested:)
+    @loa3_requested = loa3_requested
+  end
+
+  attr_reader :loa3_requested
+
+  def heading
+    helper.safe_join([I18n.t(
+      'titles.sign_up.completion_html',
+      accent: helper.content_tag(:strong, I18n.t("titles.sign_up.#{requested_loa}")),
+      app: APP_NAME
+    ).html_safe])
+  end
+
+  def title
+    I18n.t(
+      'titles.sign_up.completion_html',
+      accent: I18n.t("titles.sign_up.#{requested_loa}"),
+      app: APP_NAME
+    )
+  end
+
+  def image
+    helper.image_tag(
+      helper.asset_url("user-signup-#{requested_loa}.svg"), width: 97, alt: '', class: 'mb2'
+    )
+  end
+
+  private
+
+  def requested_loa
+    loa3_requested ? 'loa3' : 'loa1'
+  end
+
+  def helper
+    ActionController::Base.helpers
```

Comment:
@zachmargolis Awesome, TIL. It looks like `TagHelper` doesn't cover `image_tag` so I also had to add `include ActionView::Helpers::AssetTagHelper`. I could not find the proper include to cover `asset_url` though, and all of the instruction I found relating to that particular tag suggested to use the extra helper method like it's configured now, so we may still be stuck with it. Updated the PR w/ the other changes.